src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
 body {
+  font-family: $number-font;
+  font-size: $number-size;
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  box-sizing: border-box;
+  margin: 0 auto;
```

Comment:
Don't set external styles to the block. [Typical BEM mistakes](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes)